closed/src/java.base/share/classes/com/sun/crypto/provider/NativeCipherBlockChaining.java

```diff
@@ -260,6 +260,18 @@ int encrypt(byte[] plain, int plainOffset, int plainLen,
             throw new ProviderException("Internal error in input buffering");
         }
 
+        /**
+         * OpenSSL doesn't support overlapping buffers.
+         * If plaintext and ciphertext byte arrays are overlapping, make a copy of plain.
+         */
+        if (plain == cipher) {
+            byte[] copyOfInput = null;
+            copyOfInput = new byte[plainLen];
+            System.arraycopy(plain,  plainOffset, copyOfInput, 0, plainLen);
```

Comment:
Please join the declaration and the assignment:
```
    byte[] copyOfInput = new byte[plainLen];
```
Please also remove the extra space after the first comma.

Reply:
done